src/site/content/en/blog/shape-detection/index.md

```diff
@@ -0,0 +1,351 @@
+{% endAside %}
+
+The Shape Detection API is currently available as an origin trial.
+[Origin trials][ot-what-is] allow you to try out new features and give
+feedback on usability, practicality, and effectiveness to the Chrome team and the web
+standards community. For more information, see the
+[Origin Trials Guide for Web Developers][ot-dev-guide].
+
+### Register for the origin trial {: #ot }
+This conversation was marked as resolved by petele
+
+1. [Request a token][ot] for your origin.
+2. Add the token to your pages, there are two ways to provide this token on
+   any pages in your origin:
+     * Add an `origin-trial` `<meta>` tag to the head of any page. For example,
+       this may look something like: <br>
+       `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+     * If you can configure your server, you can also provide the token on pages
+       using an `Origin-Trial` HTTP header. The resulting response header should
+       look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+### Alternatives to the origin trial
+
+If you want to experiment with the Shape Detection API locally, without an
+origin trial, enable the `#enable-experimental-web-platform-features`
+flag in `chrome://flags`.
+
+
+### Using the Shape Detection API during the origin trial
+
+The interfaces of all three detectors, `FaceDetector`, `BarcodeDetector`, and
+`TextDetector`, are similar. They all provide a single asynchronous method
+called `detect()` that takes an
+[`ImageBitmapSource`](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#imagebitmapsource)
+as an input (that is, either a
+[`CanvasImageSource`](https://html.spec.whatwg.org/multipage/canvas.html#canvasimagesource), a
+[`Blob`](https://w3c.github.io/FileAPI/#dfn-Blob), or
+[`ImageData`](https://html.spec.whatwg.org/multipage/canvas.html#imagedata)).
+
+For `FaceDetector` and `BarcodeDetector`, optional parameters can be passed
+to the detector's constructor that allow for providing hints to the
+underlying native detectors.
+
+Please carefully check the support matrix in the
+[explainer](https://github.com/WICG/shape-detection-api#overview) for an
+overview of the different platforms.
+
+{% Aside 'gotchas' %}
+  If your `ImageBitmapSource` has an
+  [effective script origin](https://html.spec.whatwg.org/multipage/#concept-origin)
+  which is not the same as the   document's effective script origin, then
+  attempts to call `detect()` will fail with a new
+  `SecurityError` [`DOMException`](https://heycam.github.io/webidl/#idl-DOMException).
+  If your image origin supports CORS, you can use the
+  [`crossorigin`](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes)
+  attribute to request CORS access.
+{% endAside %}
+
+### Working with the `FaceDetector` {: #facedetector}
+
+The `FaceDetector` always returns the bounding boxes of faces it detects in
+the `ImageBitmapSource`. Dependent on the platform, more information
```

Comment:
```suggestion
the `ImageBitmapSource`. Depending on the platform, more information
```
This can be ignored if `Dependent` was intentional but it seems like `Depending` was the intended word